travis.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
if you put a
set -e
here, it will force an exit on any errors, if that helps. 

Nice to have this back.

Reply:
Good idea, I just added `set -e`